### DISCUSSION TITLE
Component lists on hosted.weblate.org?

### DISCUSSION THREAD

**Discussion by kdambekalns** [WeblateOrg/weblate#6801] [Q&A] [answered]
I found the (scarce) documentation on component lists, but they seem not to be available on hosted.weblate.org. Is that correct?

**Comment by nijel** [comment, marked as answer]
These can be currently configured only from the Django admin interface and are not available on hosted.weblate.org.

See also https://github.com/WeblateOrg/weblate/issues/1370
